### ISSUE TITLE
Flanneld not started in v0.6.0

### ISSUE THREAD

**Issue by daniel-yavorovich** [coreos/flannel#494]
Unfortunately new flannel version not started:

`/opt/bin/flanneld: error while loading shared libraries: libpthread.so.0: cannot open shared object file: No such file or directory`

**Comment by tomdee** [comment]
Can you share a little more about the environment you're running in? What ARCH do you have? Are you running in a container?

**Comment by wenerme** [comment]
Same error started by [kube-deploy](https://github.com/kubernetes/kube-deploy/blob/master/docker-multinode/common.sh#L152) (I fix the version format.)

```
/opt/bin/flanneld: error while loading shared libraries: libpthread.so.0: cannot open shared object file: No such file or directory
/opt/bin/flanneld: error while loading shared libraries: libpthread.so.0: cannot open shared object file: No such file or directory
```

**Comment by tomdee** [comment]
Yep, this is a problem :unamused: 

Caused by a late change to stop compiling the binary statically (since it caused problems with the gce backend).

Related to https://github.com/docker-library/official-images/issues/1807

**Comment by tomdee** [comment]
I've now released v0.6.1 so closing
